src/extensions/spikeamplitudes/SpikeAmplitudesView/SpikeAmplitudesPanel.ts

```diff
@@ -0,0 +1,155 @@
+import { Brush, CanvasPainter, Font, Pen } from "../../CanvasWidget/CanvasPainter"
+import { HitherContext, Recording, Sorting } from "../../extensionInterface"
+
+// const calculationPool = createCalculationPool({maxSimultaneous: 6})
+
+const colorList = [
+    'blue',
+    'green',
+    'red',
+    'orange',
+    'purple'
+]
+const colorForUnitId = (unitId: number) => {
+    while (unitId < 0) unitId += colorList.length
```

Comment:
We have negative unit ids??

Reply:
I've only seen them as non-negative, but I don't think we should exclude negatives.